### PR TITLE
 Default config items should be present on initial app start 

### LINKED ISSUE #89: [Feature Request] - Default config items should be present on initial app start
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find another request for the same or similar feature.

## Describe the solution you'd like

The app should first start with a default set of config items, which should be set in the DB if they don't exist already. We should utilize the seed functionality of Typeorm to make this happen: https://medium.com/@bansalsushil_34403/how-to-seed-typeorm-d9637a5948cc

### PR DESCRIPTION
**Pull request by smitshah2531**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes

Added new migration for default config items

## Related Issues

This PR will close #89 - Default config iteams should be present on initial app start

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,7 +90,6 @@
     "@types/express": "^4.17.1",
     "@types/jest": "^24.0.20",
     "@types/luxon": "^1.21.0",
-    "@types/node": "^11.15.0",
```
Comment (johnkahn) [inline package.json:93]: Why was this removed? I believe it's still required

File: src/migration/1600969675208-defaultConfigItems.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class defaultConfigItems1600969675208 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(
+            'INSERT INTO "configuration" ("key", "value") VALUES ("teamRegistrationActive", "true")', undefined
```
Comment (johnkahn) [inline src/migration/1600969675208-defaultConfigItems.ts:7]: @SpencerKaiser thoughts on if these should be true or false?

**Review by johnkahn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (smitshah2531) [reply package.json:93]: It was removed while I was trying to installing typeorm. I will install the `@types/node` and fix it!

File: src/migration/1600969675208-defaultConfigItems.ts (hunk 2 above)
Comment (smitshah2531) [reply src/migration/1600969675208-defaultConfigItems.ts:7]: Ya. I was confused about this too. I guess @SpencerKaiser mentioned we need to have default values disabled. So should I change it to false?

File: package.json (hunk 1 above)
Comment (smitshah2531) [reply package.json:93]: @johnkahn do we want to keep version "11.15.0"? I installed it and it is showing me version "14.11.2"

File: package.json [hunk 3]
```diff
@@ -65,6 +65,7 @@
     "@slack/types": "^1.9.0",
     "@slack/web-api": "^5.12.0",
     "@slack/webhook": "^5.0.2",
+    "@types/node": "^14.11.2",
```
Comment (SpencerKaiser) [inline package.json:68]: Revert this change; this belongs in the `devDependencies` section
```suggestion
```

File: package.json (hunk 1 above)
Comment (SpencerKaiser) [reply package.json:93]: @johnkahn it was moved to `dependencies`

@smitshah2531 unless there was a reason for the version bump, let's handle that in a separate issue. Just revert this change back to the original.

**Review by SpencerKaiser — CHANGES_REQUESTED** [review]
Two quick changes and then we can merge!

File: package.json (hunk 1 above)
Comment (SpencerKaiser) [inline package.json:93]: You need to add this dependency back. Also make sure to fix the associated changes in `package-lock.json` (just revert back to what they were.

The end state is that the `Files changed` tab here shouldn't include either `package.json` or `package-lock.json`

File: package.json [hunk 4]
```diff
@@ -117,4 +117,4 @@
     "ts-jest": "^24.0.0",
     "ts-node": "^8.4.1"
   }
-}
+}
```
Comment (SpencerKaiser) [inline package.json:120]:
```suggestion
}

```

File: package-lock.json [hunk 5]
```diff
@@ -15953,4 +15953,4 @@
       }
     }
   }
-}
+}
```
Comment (SpencerKaiser) [inline package-lock.json:15956]:
```suggestion
}

```

**Review by SpencerKaiser — APPROVED** [review]
🚢
